testdata/scripts/octal-constants.txt

```diff
@@ -0,0 +1,20 @@
+gofumpt -w .
+cmp foo.go foo.go.golden
+
+-- foo.go --
+package p
+
+const (
+	i = 0
+	j = 022
+	k = 0o_7_5_5
```

Comment:
Add potential false positives too, like `1022` and `0229`. Neither of those should match, but it might be the case if the regex doesn't properly require matching the entire string.

Reply:
Done for `1022`.

If I add `0229` the test fails with `foo.go:8:9: invalid digit '9' in octal literal`. Rather than hacking the tests to allow this error, I think it's ok to assume that the `[0-7_]` part of the regexp is correct.

Reply:
Ah, right, I forgot that it's an invalid literal.